keyboards/handwired/drevogramr_qmk/config.h

```diff
@@ -0,0 +1,181 @@
+/* Copyright 2021 wwwMADwww <4d4144h@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xD2E0
+#define PRODUCT_ID      0x62A2
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Handwire
+#define PRODUCT         Drevo Gramr QMK
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 16
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+// does not matter in custom matrix scanning mode, just for information
+#define DIODE_DIRECTION ROW2COL
+
+
+#define LAYER_STATE_8BIT
+
+
+/*
+    pins for 74hc595 shift registers
+    https://www.ti.com/lit/ds/symlink/sn74hc595.pdf
+*/
+#define MATRIX_PIN_REG_DATA     D1  /* SER   / SI  / DS   */
+#define MATRIX_PIN_REG_SHIFT    D0  /* SRCLK / SCK / SHCP */
+#define MATRIX_PIN_REG_STORE    D4  /* RCLK  / RCK / STCP */
+
+/* 
+    pins for row input 
+    first = bottom, last = top
+*/
+#define MATRIX_PIN_ROWS { E6, B4, B5, B6, B7, D6 }
+
+
+/* Drevo Gramr PCB allows dynamic scanning of backlight LEDs, not just full board backlighting */
+#define BACKLIGHT_PIN         C6
+#define BACKLIGHT_LEVELS      4
+#define BACKLIGHT_ON_STATE    0
+#define BACKLIGHT_LIMIT_VAL   255
+#define BREATHING_PERIOD      6
+#define BACKLIGHT_BREATHING
+
+#define LED_NUM_LOCK_PIN     B2
+#define LED_CAPS_LOCK_PIN    D3
+#define LED_SCROLL_LOCK_PIN  D2
+#define LED_GUI_LOCK_PIN     D7
+#define LED_PIN_ON_STATE     0
+#define LED_PIN_OFF_STATE    (LED_PIN_ON_STATE ^ 1)
```

Comment:
This isn't used.
```suggestion
```

Reply:
It is used here

https://github.com/wwwMADwww/qmk_firmware/blob/28fcc16961d7c4e03b00b639541df6755e18c013/keyboards/handwired/drevogramr_qmk/drevogramr_qmk.c#L135-L137

https://github.com/wwwMADwww/qmk_firmware/blob/28fcc16961d7c4e03b00b639541df6755e18c013/keyboards/handwired/drevogramr_qmk/drevogramr_qmk.c#L167-L173